react-16/src/FilterList.js

```diff
@@ -0,0 +1,34 @@
+import React, {useState} from 'react';
+
+const ShowList = (props) => {
+  const {list, text} = props;
+
+  return (
+      <>
+        {list.filter(item => item.toLowerCase().startsWith(text.toLowerCase())).map(item => <p key={item}>{item}</p>)}
```

Comment:
This line is unreadable.
`.filter` should be outside of the JSX and only `.map` should be inside.
Also, no need for `startsWith`

```js
const items = list.filter(item => item.toLowerCase() === text.toLowerCase())
...
<>
{items.map(item => <p key={item}>{item}</p>)}
</>
```